randrctl/profile.py

```diff
@@ -167,6 +167,15 @@ def find_best(self, available_profiles: list, xrandr_outputs: list):
             score = self._calculate_profile_score(p, xrandr_outputs)
             if score >= 0:
```

Comment:
what do you think about changing this to `> 0` ?

Honestly, I do not remember, why it is `>= 0` and I haven't touched code for quite a while, so you might know better now.

But at least in the context of the feature you implemented listing profiles with score 0 doesn't seem to have any sense.

Reply:
If I'm right, a profile scored 0 means its doesn't have any match rules and it will always match. I'm not sure about what is the scenario people use a profile without any match rules.  So `s > 0` sounds more sense than `s >= 0` in this case. I will change it to `s > 0` later.